content/en/logs/archives/rehydrating.md

```diff
@@ -103,13 +100,26 @@ In order to rehydrate log events from your archives, Datadog uses the IAM Role i
 
 #### Adding role delegation to S3 archives
 
-Datadog only supports rehydrating from archives that have been configured to use role delegation to grant access. Once you have modified your Datadog IAM role to include the IAM policy above, ensure that each archive in your [archive configuration page][10] has the correct AWS Account + Role combination.
+Datadog only supports rehydrating from archives that have been configured to use role delegation to grant access. Once you have modified your Datadog IAM role to include the IAM policy above, ensure that each archive in your [archive configuration page][3] has the correct AWS Account + Role combination.
 
 {{< img src="logs/archives/log_archives_rehydrate_configure_s3.png" alt="Adding role delegation to S3 archives" responsive="true" style="width:75%;">}}
 
-### Feedback
+[1]: https://app.datadoghq.com/account/settings#integrations/amazon-web-services
+[2]: /integrations/amazon_web_services/?tab=allpermissions#installation
+[3]: https://app.datadoghq.com/logs/pipelines/archives
+
+{{% /tab %}}
+
+{{% tab "Google Cloud Storage" %}}
+
+In order to rehydrate log events from your archives, Datadog uses a Service Account with the Storage Object Viewer role. You can grant this role to your Datadog Service Account from the [GCP IAM Admin page][1] by editing the Service Account's permissions, adding another role, and then selecting Storage > Storage Object Viewer. 
```

Comment:
```suggestion
In order to rehydrate log events from your archives, Datadog uses a service account with the Storage Object Viewer role. You can grant this role to your Datadog service account from the [GCP IAM Admin page][1] by editing the service account's permissions, adding another role, and then selecting Storage > Storage Object Viewer. 
```

Reply:
cool -- updated this in my latest commit, thanks!